### PR TITLE
 #1 - project skeleton

### LINKED ISSUE #1: Project skeleton
Start a new Flutter project using builtin skeleton project template.

### PR DESCRIPTION
**Pull request by Lee0z**
it should close #1.

### CODE REVIEW

File: analysis_options.yaml
Comment (AleksandraKozubal) [inline analysis_options.yaml:1]: We can delete that, workflows are created in another pr

**Review by EwelinaSkrzypacz — COMMENTED** [review]
Please add in `readme.md` file instruction, how to run project 

Simple instruction like:

`flutter doctor`
`flutter pub get`
`flutter run`

will be enough.

File: analysis_options.yaml
Comment (Lee0z) [reply analysis_options.yaml:1]: Isn't this file necessary to lint the code?

Comment (AleksandraKozubal) [reply analysis_options.yaml:1]: You're right, this one is used in static analysis, which isn't mentioned in this separated pr.

File: readme.md [hunk 1]
```diff
@@ -1 +1,18 @@
 ## e-motion-mobile
+
+### Local development
+
+Download necessary dependencies (packages):
+```shell
+flutter pub get
+```
+
+Check flutter installation:
```
Comment (EwelinaSkrzypacz) [inline readme.md]: I think that `flutter doctor` should be first and after that `flutter pub get`

Comment (Lee0z) [reply readme.md]: Good idea

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by AleksandraKozubal — APPROVED** [review]

**Pull request merged**